src/gui/mainwindow.cpp

```diff
@@ -205,6 +218,11 @@ MainWindow::MainWindow(QWidget *parent)
     m_tabs = new HidableTabWidget(this);
     connect(m_tabs, SIGNAL(currentChanged(int)), this, SLOT(tabChanged(int)));
 
+#ifdef Q_OS_MAC
+    // Increase top spacing to avoid tab overlapping
+    m_ui->centralWidgetLayout->addSpacing(8);
```

Comment:
there is `m_ui->centralWidgetLayout->addWidget(m_tabs);` below, is it possible to move this block to there?